File: src/contest/winter_2022/jan_16/g/Main.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
  int n;
  cin >> n;
  int ls[n];
  for (int i = 0; i < n; i++) {
    cin >> ls[i];
  }
  sort(ls, ls + n);
  int l = (n >> 1) - 1;
  int r = n - 1;
  int cnt = n;
  while (l >= 0) {
    if (ls[r] >= ls[l] * 2) {
      l--, r--;
      cnt--;
    } else {
      l--;
    }
  }
  cout << cnt << endl;
  return 0;
}

File: src/contest/winter_2022/jan_16/g/Main.py
# https://nanti.jisuanke.com/t/A2240
n = int(input())
ls = list(map(int, input().split()))
ls.sort()
l = (n >> 1) - 1
r = n - 1
cnt = n
while l >= 0:
    if ls[r] >= ls[l] * 2:
        l -= 1
        r -= 1
        cnt -= 1
    else:
        l -= 1
print(cnt)